MvvmCross.Tests/MvvmCross.Test/MvvmCross.Test.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Why add this back in?

Reply:
This is a new Test project with shared test stuff. I don't think it should be in the core `MvvmCross` project.